administration/src/bp-modules/applications/utils/verificationHelper.tsx

```diff
@@ -0,0 +1,4 @@
+export const isEmailValid = (email: string): boolean => {
```

Comment:
```suggestion
export const isEmailValid = (email: string): boolean => /^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(email)
```
🔧 Since you are not reusing the pattern, this one liner should be enough